src/shared/components/apps.tsx

```diff
@@ -24,13 +25,12 @@ export class Apps extends Component<any, any> {
           <p>{i18n.t("choose_from_apps")}</p>
 
           <div class="row">
-            <div class="card col-6">
-              <AppDetails
+            <div class="card col-6 col-4-md">
+              <CompactAppDetails
                 name="Jerboa"
                 description="A native Android app made by Lemmy's developers"
                 link="https://github.com/dessalines/jerboa"
                 icon="/static/assets/images/jerboa.svg"
-                banner="/static/assets/images/jerboa_screen.webp"
```

Comment:
It takes up a lot of space. Removing it will allow listing many more of the lemmy apps that exist, that'll be more useful to users than the banner.

Reply:
I also think that we should keep the screenshot to give people an idea what the app looks like. The name, icon and short description are not much to go by.

Reply:
That's fair, but don't you think this layout will become unwieldy and difficult to navigate once 10-15 apps are listed? On desktops, a row of two apps takes up a screen-full. On mobile each card takes up a screen.

To me, the missing screenshot is worth what will be gained in being able to view more of the existing apps comfortably. Mastodon lists clients similarly: https://joinmastodon.org/apps

I'll close the PR if you don't think this will be an issue. @Nutomic @dessalines

Reply:
Yeah I think that's not too much of an issue. Especially since we've got a whole joinlemmy-site redesign coming up.